Kickstarter-iOS/Views/CommentInputTextView.swift

```diff
@@ -0,0 +1,51 @@
+import Library
+import Prelude
+import UIKit
+
+final class CommentInputTextView: UITextView {
+  // MARK: - Properties
+
+  let maxHeight: CGFloat = 200
```

Comment:
How is this value determined?

Reply:
It is part of the acceptance criteria for the ticket